tmp/lanelet2_extension/lib/query.cpp

```diff
@@ -50,6 +49,21 @@
 
 using lanelet::utils::to2D;
 
+namespace
```

Comment:
@esteve 
Thank you for the comment.
We will replace the autoware_utils with the tier4_autoware_utils as follows. tire4_autoware_utils also has `normalizeRadian`, and I will remove the implementation of lanelet2_extension later once tier4_autoware_utils is moved to autoware_common.
https://github.com/orgs/autowarefoundation/discussions/4855

Reply:
@esteve 
I moved tier4_autoware_utils in autoware.universe to autoware_utils in autoware_common. Does this seem good to you?
Then I will try `git filter-repo` to move the history as well.

Reply:
I don't want to work on fixing the function's format of the snake_case now since it requires a lot of changes including autoware.universe. After this PR is merged, we can fix autoware_utils and also autoware.unvierse where the functions are used.

Reply:
Ah, tier4_autoware_uils depends on `tier4_planning_msgs`. It may be better to remove it.

Reply:
I'd move all the code from `tier4_autoware_utils` that doesn't depend on any `tier4_` package (e.g.  tier4_debug_msgs and tier4_planning_msgs) and then we can discuss what to do with the remaining code in `tier4_autoware_utils`.

@xmfcx @mitsudome-r is it ok if `autoware_utils` depends on message packages (like `autoware_planning_msgs`) ?

Reply:
> is it ok if autoware_utils depends on message packages (like autoware_planning_msgs) ?

Yes, that's alright.